File: webapi/FFischbach.Events.API/Models/Participant.cs
namespace FFischbach.Events.API.Models
{
    public class Participant
    {
        public int Id { get; set; }
        public required byte[] EncryptedData { get;set; }
        public bool IsContact { get; set; }
        public bool? VIP { get; set; }
        public int GroupId { get; set; }
        public Group? Group { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: webui/src/models/in/Participant.ts
export class Participant {
  constructor(
    public id: number,
    public Email: string,
    public FirstName: string,
    public LastName: string,
    public BirthDate: string,
    public vip: boolean,
    public createdAt: string,
    public encryptedData?: string,
  ) {}
}
